Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ManipulationHandler.cs

```diff
@@ -466,6 +465,37 @@ private void InvokeStateUpdateFunctions(State oldState, State newState)
         }
         #endregion Private Methods
 
+        #region Public Methods
+
+        /// <summary>
+        /// Releases the object that is currently manipulated
+        /// </summary>
+        public void ForceEndManipulation()
+        {
+            // release rigidbody and clear pointers
+            ReleaseRigidBody();
+            pointerIdToPointerMap.Clear();
+
+            // end manipulation
+            State newState = State.Start;
+            InvokeStateUpdateFunctions(currentState, newState);
+            currentState = newState;
+        }
+
+        /// <summary>
+        /// Gets the grab point for the given pointer id
```

Comment:
Or zero if none is found?

Reply:
Perhaps we should assert here that the pointer exists in the map, returning zero if none is found might lead to some tricky to discover bugs in the future.